sys/arm64/arm64/copyinout.S

```diff
@@ -282,8 +278,10 @@ ENTRY(copyincap)
 
 	/* Create a capability from kdaddr */
 	mov	x5, x1			/* Backup kaddr */
+#ifndef __CHERI_PURE_CAPABILITY__
 	cvtd	c1, x1			/* Store through a capability */
 	scbnds	c1, c1, x2		/* Set the bounds */
+#endif
```

Comment:
This confuses me somewhat. Shouldn't x5 always be the same as x1 given we run with a zero-base DDC even for a hybrid kernel? But certainly in a purecap kernel we shouldn't need to backup kdaddr.

Reply:
Assuming the zero-base DDC we can probably ditch x5 and always use c/x1